scss/_deprecated.scss

```diff
@@ -1,3 +1,31 @@
+/// Button custom theme construct
+/// Outputs styles for all button states for custom buttons based on a background color and an accent color
+///
+/// @example
+/// .my-button--custom {
+/// 	@include oButtonsTheme(black, paper);
```

Comment:
Is this example meant for `oButtonsTheme` or `oButtonsCustomTheme`?

Reply:
Good spot. Either way it's wrong. I'll update to `oButtonsCustomTheme`.